operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -608,12 +608,11 @@ function diff_model() {
   export __WLSDEPLOY_STORE_MODEL__=1
   # $1 - new model, $2 original model
 
-  # no need to redirect output, -output_dir for compareModel will generate the output file
-  ${WDT_BINDIR}/compareModel.sh -oracle_home ${ORACLE_HOME} -output_dir /tmp $1 $2 > /dev/null 2>&1
+  ${WDT_BINDIR}/compareModel.sh -oracle_home ${ORACLE_HOME} -output_dir /tmp $1 $2 >  ${WDT_OUTPUT} 2>&1
```

Comment:
Is the double space after the > intentional?

Reply:
no, I can remove it.